rviz_default_plugins/test/rviz_default_plugins/transformation/frame_transformer_tf_test.cpp

```diff
@@ -45,8 +45,12 @@ class FrameTransformerTfFixture : public testing::Test
   FrameTransformerTfFixture()
   {
     auto clock = std::make_shared<rclcpp::Clock>(RCL_ROS_TIME);
+    if (!rclcpp::is_initialized()) {
+      rclcpp::init(0, nullptr);
```

Comment:
If we call `rclcpp::init`, we should also call `rclcpp::shutdown` when we're finished.

I would add the static fixture methods:

```c++
  static void SetUpTestCase()
  {
    rclcpp::init(0, nullptr);
  }

  static void TearDownTestCase()
  {
    rclcpp::shutdown();
  }
```